docs/components/card.md

```diff
@@ -995,3 +995,46 @@ Card columns can also be extended and customized with some additional code. Show
   }
 }
 {% endhighlight %}
+
+## Publication
+Shows implementations of the publication element.
+
+{% example html %}
+<div class="card">
+  <article class="card-block card-block-action publication-block has-arrow">
+    <header class="publication-header">
+      <a class="publication-title" href="#" title="Personalsachbearbeiter/in in Grünwald" itemprop="title">Lorem ipsum dolor sit amet, consetetur sadipscing elitr</a>
```

Comment:
Yeah I thought about this as well while writing the previous comment, at first I didn't like the microformats at all in our styleguide, but then I noticed some spans further down which wouldn't exist if it weren't for the microformats so maybe it's not a bad idea to include them here.